orbs/helm/Dockerfile.deploy

```diff
@@ -4,17 +4,19 @@ LABEL vendor="JobTeaser"
 LABEL maintainer="opensource@jobteaser.com"
 
 ENV HELM_VERSION 2.12.3
-ENV KUBE_VERSION 1.14.5
+ENV KUBE_VERSION 1.14.10
 
 ENV DEBIAN_FRONTEND noninteractive
 
+COPY circle-wait-job /bin/circle-wait-job
+
 RUN apt-get update -y && \
     apt-get upgrade -y && \
     apt-get install -y --no-install-recommends \
       # base
       wget ca-certificates \
       # checkout
-      git openssh-client \
+      git openssh-client ruby \
```

Comment:
You could have put that one below a new comment indicating it's for `circle-wait-job` ;)